src/main/js/utils/Grade.js

```diff
@@ -0,0 +1,72 @@
+import queryString from 'query-string';
```

Comment:
I've only capitalized the file name for files that are actual classes, lowercase for utility like this file

Reply:
Done.